spec/exceptions_spec.rb

```diff
@@ -1,9 +0,0 @@
-require 'spec_helper'
-require 'dentaku/exceptions'
-
-describe Dentaku::UnboundVariableError do
-  it 'includes variable name(s) in message' do
-    exception = described_class.new(['length'])
-    expect(exception.message).to match /length/
```

Comment:
Do we still test that the `UnboundVariableError` message includes the variable name(s)?